plugins/admin/core/root.go

```diff
@@ -31,21 +31,17 @@ import (
 var cfgFile string
 
 // rootCmd represents the base command when called without any subcommands
-
 func NewAdminCommand(params ...pkg.AdminParams) *cobra.Command {
 	p := &pkg.AdminParams{}
 	p.Initialize()
 
 	rootCmd := &cobra.Command{
 		Use:   "kn\u00A0admin",
 		Short: "A plugin of kn client to manage Knative",
-		Long: `kn admin: a plugin of kn client to manage Knative for administrators.
+		Long:  `kn admin: a plugin of kn client to manage Knative for administrators`,
```

Comment:
@maximilien I don't know why there is aa unicode character, @zhanggbj if no special reason here, I will replace it with space. 
@navidshaikh I think line ```Long:``` is formatted by **GoFmt**, I tried, if there are multiple string lines for ```Long:```, GoFmt will not add extra space, but if there is a single line string for it, GoFmt will add extra spaces to make it align with the above lines. So I will keep it.

Reply:
@chaozbj @maximilien the unicode here is a non-break space to show usage as below, so I think we need it.
```
Usage:
  kn admin [command]
```
Without the unicode, it will be
```
Usage:
  kn [command]

```

Reply:
@zhanggbj ok, I will keep it.

Reply:
@zhanggbj @chaozbj: I tried removing `\u00A0a` and building locally, I see the usage as expected `  kn admin [command]`.